content/en/docs/setup/upgrade/helm/index.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Upgrade with Helm
+linktitle: Upgrade with Helm
+description: Upgrade and configure Istio for in-depth evaluation.
+weight: 27
+keywords: [kubernetes,helm]
+owner: istio/wg-environments-maintainers
+icon: helm
+test: no
+---
+
+Follow this guide to upgrade and configure an Istio mesh using
+[Helm](https://helm.sh/docs/) for in-depth evaluation.  This guy assumes you have already performed an
```

Comment:
The change from guy to guide was missed.

Reply:
Fixed